### PR TITLE
Add ROS Official Docker image 

### PR DESCRIPTION
**Pull request by ruffsl**
Add ROS Official Docker image documentation

### CODE REVIEW

**Comment by ruffsl** [comment]
Awesome, looks like the [PR#751](https://github.com/docker-library/official-images/pull/751#event-320605576) for the official-images is now merged.
Let me know if we need any changes to the respective docs.
Pinging @yosifkit and @tianon

**Comment by tianon** [comment]
Can you please rebase and force push?

```console
$ git pull --rebase https://github.com/docker-library/docs.git
$ git push -f
```

You might also want to check out the output of `./markdownfmt.sh -d ros/content.md`.

File: ros/content.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+[Blog][@ROS_Blog]: Stay up-to-date  
+[OSRF][@OSRF]: Open Source Robotics Foundation  
+
+[@ROS.org]:     http://www.ros.org/ "Main ROS website"
+[@ROS_Wiki]:    http://wiki.ros.org/ "Find tutorials and learn more"
+[@ROS_Answers]: http://answers.ros.org/questions/ "Ask questions. Get answers"
+[@ROS_Blog]:    http://www.ros.org/news/ "Stay up-to-date"
+[@OSRF]:        http://www.osrfoundation.org/ "Open Source Robotics Foundation"
+
+[@ROS_Logo]:    https://raw.githubusercontent.com/docker-library/docs/master/ros/logo.png "ROS Logo"
```
Comment (tianon) [inline ros/content.md]: What's the purpose of making the logo itself a link to the logo?

**Comment by ruffsl** [comment]
Rebased!

**Comment by ruffsl** [comment]
Ok, I had to cut out some fancy markdown linking for Travis to pass.
Should be good now!

**Comment by tianon** [comment]
LGTM

**Pull request merged**